html/admin/index.html

```diff
@@ -0,0 +1,195 @@
+      <p class="lead">Configure the channel used by the Wireless Access Point</p>
+          <form class="simple_form form-inline" id="form_channel">
+              <div class="form-group required"><label class="required sr-only" for="channel"><abbr title="required">*</abbr> Channel</label><input class="string required form-control" id="input_channel" name="channel" placeholder="Enter Channel" type="text"></div>
+
+              <input class="btn btn-default" name="update_channel" type="submit" value="Update">
+              <div style="display:none" id="update_channel_success">✓</div>
+              <div style="display:none" id="update_channel_failure">&times;</div>
+        </form>
+    </div>
+
+    <div class="container" id="hostname" style="display: none;">
+      <div class="page-header">
+        <h1>Web Server</h1>
+      </div>
+      <p class="lead">Web server hostname</p>
+          <form class="simple_form form-inline" id="form_hostname">
+              <div class="form-group required"><label class="required sr-only" for="hostname"><abbr title="required">*</abbr> Hostname</label><input class="string required form-control" id="input_hostname" name="hostname" placeholder="Enter Hostname" type="text"></div>
+
+              <input class="btn btn-default" name="update_hostname" type="submit" value="Update">
+              <div style="display:none" id="update_hostname_success">✓</div>
+              <div style="display:none" id="update_hostname_failure">&times;</div>
+        </form>
+    </div>
+
+    <div class="container" id="system" style="display: none;">
+      <div class="page-header">
+        <h1>System</h1>
+      </div>
+      <p class="lead">System Functions</p>
+        <form class="simple_form form-inline" id="form_unmountusb">
+            <input class="btn btn-default" name="unmountusb" type="submit" value="Unmount USB">
+            <div style="display:none" id="unmountusb_success">✓</div>
+            <div style="display:none" id="unmountusb_failure">&times;</div>
+        </form>
+
+        <form class="simple_form form-inline" id="form_shutdown">
+            <input class="btn btn-default" name="shutdown" type="submit" value="Shutdown">
+            <div style="display:none" id="shutdown_success">✓</div>
+            <div style="display:none" id="shutdown_failure">&times;</div>
+        </form>
+
+        <form class="simple_form form-inline" id="form_reboot">
+            <input class="btn btn-default" name="reboot" type="submit" value="Reboot">
+            <div style="display:none" id="reboot_success">✓</div>
+            <div style="display:none" id="reboot_failure">&times;</div>
+        </form>
+
+        <form class="simple_form form-inline" id="form_reset">
+            <input class="btn btn-default" name="reset" type="submit" value="Reset">
+            <div style="display:none" id="reset_success">✓</div>
+            <div style="display:none" id="reset_failure">&times;</div>
+        </form>
+    </div>
+
```

Comment:
No reason in particular. Since we are just using basic authentication, logging out is a little tricky. See this http://stackoverflow.com/a/233551/290918. Not sure whether or not it makes sense to have that or not. I guess there might be a scenario where the biblebox is administered from a shared machine. We may need to move authentication back into the php application.